oxide/lib.go

```diff
@@ -118,6 +155,78 @@ func defaultUserAgent() string {
 	return fmt.Sprintf("oxide.go/%s", version)
 }
 
+func profileCredentials(cfg Config) (*hostCreds, error) {
+	configDir := cfg.ConfigDir
+	if configDir == "" {
+		homeDir, err := os.UserHomeDir()
+		if err != nil {
+			return nil, fmt.Errorf("unable to find user's home directory: %w", err)
+		}
+		configDir = filepath.Join(homeDir, ".config", "oxide")
```

Comment:
Let's make `".config"`, `"config.toml"`, and `"credentials.toml"`  constants

Reply:
Done in 3d0217bebd5db2bff5e84dbb47ac604b8742195e